subprojects/packagefiles/openblas/lapack-netlib/BLAS/SRC/meson.build

```diff
@@ -276,9 +276,18 @@ if prec == 's' or build_single or build_all_prec
 endif
 
 # Create the blas library
-netlib_blas = library(_netlib_blas_name,
-               sources: _blas_netlib_srcs,
-               link_with: _allblas,
-               version: lapack_version,
-               soversion: lapack_major_version,
-               install: true)
+if as_subproject
+  # Avoid creation versioned .so symlinks. We can't pass empty string or list
+  # to the version/soversion keyword to make this more concise.
```

Comment:
It's possible to pass additional `kwargs: ...` as a dictionary to a function, which allows conditionally passing arguments.

Reply:
Ah thanks, that was exactly what I was looking for.

Should `kwargs` be in the docs? https://mesonbuild.com/Reference-manual_functions.html#library and other functions don't have this keyword.

Reply:
It is described at https://mesonbuild.com/Syntax.html#function-calls

This is baked into the function call syntax, individual functions aren't aware that such a keyword exists and indeed the type checking for `library('abc', 'abc.c', kwargs: {'c_args': '-DFOO=1'})` will literally believe that the function was called as `library('abc', 'abc.c', c_args: '-DFOO=1')`